draft-ietf-acme-acme.md

```diff
@@ -1207,9 +1315,14 @@ authorized).
 
 It is up to the server's local policy to decide which names are acceptable in a
 certificate, given the authorizations that the server associates with the
-client's account key.  A server MAY consider a client authorized for a wildcard
-domain if it is authorized for the underlying domain name (without the "*"
-label).  Servers SHOULD NOT extend authorization across identifier types.  For
+client's account key.  For example, a server might have a policy that it will
+consider a client authorized for a wildcard domain if it is authorized for some
+set of names that match the wildcard.  If the client does not yet have
```

Comment:
It seems dangerous to hint at the right requirements without actually stating them explicitly or providing any guidance for prospective server implementers.  Getting these requirements wrong would allow the incorrect issuance of wildcard certificates.

For example, it doesn't suffice to generate N random subdomains and require that the client match those, without also requiring that the same client (with the same key) prove ownership of the containing domain (e.g. proving ownership of example.org in addition to random examples of *.example.org).  Otherwise, anyone could obtain a wildcard certificate for any site that allows user-created subdomains with arbitrary names and user-controlled content, by manually creating the requested names.  Requiring proof of ownership of the